app/controllers/subscribers_auth_token_controller.rb

```diff
@@ -3,6 +3,8 @@ class SubscribersAuthTokenController < ApplicationController
 
   def auth_token
     subscriber = Subscriber.find_by_address!(expected_params[:address])
+    head :forbidden and return if subscriber.linked_to_govuk_account?
```

Comment:
We've used this patten everywhere and I generally like it.
It's pretty human readable.

But it just occured to me, i'm not sure if that's just a style preference or is there actually a difference when compared to:


```
return head :forbidden if subscriber.linked_to_govuk_account?
```

Remind me how we ended up using this so much? As I recall it was somthing neat you or Bruce found?

Reply:
I think Bruce came up with it.  I personally find it more readable than `return head :forbidden`, because that to me reads like it should be `return(head) :forbidden` which would be a syntax error... but Ruby actually seems to handle it just fine.

Reply:
ok great, so both work but we prefer this for readability.

I agree, it's nice! just wanted t remind myself why we'd standarised around it!